_docs/reference/istioctl-autogen/istioctl.md

```diff
@@ -0,0 +1,37 @@
+---
+category: Reference
+title: Istioctl
+bodyclass: docs
```

Comment:
Is there an issue tracking moving the content over from the .md to the .go file?